src/wwwroot/partials/settings/billing.html

```diff
@@ -200,8 +200,8 @@ <h3 class="special">{{ 'payment_method_title' | translate }}</h3>
       <div class="detached--error-container" ng-class="vm.submitted && vm.paymentFailure ? 'show':'hide'">
         <span class="detached--error">{{'validation_error_payment_failure' | translate}}</span>
       </div>
-      <button class="button button--small button--white modify-button" ng-click="vm.showConfirmation = false;vm.paymentFailure = false;">{{'billing_change_info_button' | translate}}</button>
-      <button class="button button--small buy-button" ng-click="vm.submitBillingPayment()">{{'buy_text' | translate}}</button>
+      <button class="button button--small button--white modify-button" ng-click="vm.cancelAction()"><span ng-if="!vm.downgrade">{{'billing_change_info_button' | translate}}</span><span ng-if="vm.downgrade">Volver a My Plan</span></button>
+      <button class="button button--small buy-button" ng-click="vm.submitBillingPayment()"><span ng-if="!vm.downgrade">{{'buy_text' | translate}}</span><span ng-if="vm.downgrade">Confirmar Downgrade</span></button>
```

Comment:
We should add translations for "Volver a My Plan" and "Confirmar Downgrade" texts.

Reply:
Good Catch!